pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -956,3 +957,58 @@ func getKey(config *mcfgv1.ContainerRuntimeConfig, t *testing.T) string {
 	}
 	return key
 }
+
+// TestCleanUpDuplicatedMC test the function removes the MC from the MC list
+// if the MC is of old version.
+func TestCleanUpDuplicatedMC(t *testing.T) {
```

Comment:
@QiWang19 can you please add similar tests for kubeletconfig

Reply:
test added.